### PR TITLE
OPSEXP-2519 Bump supported matrix for acs 23.2 release 

### PR DESCRIPTION
**Pull request by pmacius**
!Ref OPSEXP-2519

### CODE REVIEW

**Comment by pmacius** [comment]
We are still waiting for some of the releases but we can save some time if you take a quick look at this and then merge it when releases will be on the shelf 😃

File: deployments/values/supported-matrix.yaml [hunk 1]
```diff
@@ -92,11 +92,11 @@ matrix:
...
-      version: "~3.0.0"
+      version: "~3.1.0"
     trouter:
-      version: "~4.0.0"
+      version: "~4.1.0"
     sfs:
-      version: "~4.0.0"
+      version: "~4.1.0"
     tengine-aio:
       version: &tengine_ga_version "~5.0.0"
```
Comment (alxgomz) [inline deployments/values/supported-matrix.yaml:101]: Needs a bump to 5.1.x (tengine is what is refered to as T-Core in PRODDEL)

File: deployments/values/supported-matrix.yaml [hunk 2]
```diff
@@ -77,13 +77,13 @@ matrix:
       version: "4.0"
       pattern: *ga_hotfixes_pattern
     sync:
-      version: "4.0"
+      version: "5.0"
```
Comment (alxgomz) [inline deployments/values/supported-matrix.yaml:80]: We're not releasing a GA during this train so we have to stick to 4.0 for now

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: deployments/values/supported-matrix.yaml [hunk 3]
```diff
@@ -41,11 +41,11 @@ matrix:
...
       pattern: *development_pattern
     intelligence:
-      version: ">=3.0.0-0"
+      version: ">=3.1.0-0"
     trouter:
-      version: ">=4.0.0-0"
+      version: ">=4.1.0-0"
     sfs:
-      version: ">=4.0.0-0"
+      version: ">=4.1.0-0"
```
Comment (gionn) [inline deployments/values/supported-matrix.yaml:48]: not really necessary, we can even put `>=0.0.0-0` here if you think is more clear or keep as it is

File: deployments/values/supported-matrix.yaml [hunk 4]
```diff
@@ -62,13 +62,13 @@ matrix:
...
     acs:
-      version: &acs_ga_version "23.1"
+      version: &acs_ga_version "23.2"
       pattern: *ga_hotfixes_pattern
     activemq:
       version: "5.18"
       pattern: *ga_activemq_pattern
     share:
-      version: "23.1"
+      version: "23.2"
```
Comment (gionn) [inline deployments/values/supported-matrix.yaml:71]: you can reuse the *acs_ga_version anchor, it's probably a miss when we introduced it

**Review by alxgomz — APPROVED** [review]

**Review by gionn — APPROVED** [review]

**Pull request merged**